lib/search.js

```diff
@@ -17,28 +17,47 @@ module.exports = new class LibSearch {
     }
 
     optimizeQueryTerms(input) {
+        // parse
+        let normalize = this.normalizeProt();
+        input.imdbid = input.imdbid && parseInt(input.imdbid.toString().replace('tt', ''), 10)
+        input.season = input.season && parseInt(input.season)

```

Comment:
Should not be sent to the server, the conditional that checks if season has value still giving false when the property is undefined
If season is undefined, should not enter this if
 if (input.season && input.episode) {
 -                    output[i].season = input.season.toString()